react/package.json

```diff
@@ -1,9 +1,11 @@
 {
   "devDependencies": {
     "@types/node": "^11.13.4",
+    "@types/query-string": "5",
     "@types/react": "^16.8.13"
   },
   "dependencies": {
+    "query-string": "5",
```

Comment:
Good choice hehe
https://github.com/vtex-apps/store/pull/395

Reply:
hahahah
I have to admit that I almost fell into the trap